app/components/exhibits/search_tips_link_component.html.erb

```diff
@@ -0,0 +1,8 @@
+<div class="navbar-nav searchtips-link">
+  <a class="nav-link" data-blacklight-modal="trigger" href="/search_tips">
+    <span class="mr-1 pt-0">
+      <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-info-circle" viewBox="0 0 16 16">
+        <path d="M8 15A7 7 0 1 1 8 1a7 7 0 0 1 0 14m0 1A8 8 0 1 0 8 0a8 8 0 0 0 0 16"/>
+        <path d="m8.93 6.588-2.29.287-.082.38.45.083c.294.07.352.176.288.469l-.738 3.468c-.194.897.105 1.319.808 1.319.545 0 1.178-.252 1.465-.598l.088-.416c-.2.176-.492.246-.686.246-.275 0-.375-.193-.304-.533zM9 4.5a1 1 0 1 1-2 0 1 1 0 0 1 2 0"/>
+      </svg></span><%= t('search_tips.title') %></a>
```

Comment:
There's an info icon component already in spotlight that could be used via `blacklight_icon('info')`. Should we use that icon or override it locally with this one?

Reply:
Happy to use an existing one. I'll try that out, thanks.

Reply:
I set one up using the blacklight icon method. This also changed some styling (there's an extra space that the component adds within the blacklight icon next to the svg element)